README.md

```diff
@@ -1,2 +1,49 @@
 # observability-data-model
 Documentation for telemetry data model and semantic conventions
+
+## Logging
```

Comment:
Could we move this to a seprate file e.g. `logging.md` ?

Reply:
Will do